.travis.yml

```diff
@@ -1,11 +1,11 @@
-jobs:
-  include:
-    - stage: test
-      script:
-        - docker build --target all_tests -t test .
-        - docker run --name todo-app-test --env DB_USER --env DB_PASS --env DB_NAME test
-deploy:
-  provider: script
-  script: bash ci/deploy.sh
-  on:
-    branch: main
+jobs:
+  include:
+    - stage: test
+      script:
+        - docker build --target all_tests -t test .
+        - docker run --name todo-app-test --env DB_USER --env DB_PASS --env DB_NAME --env DB_HOST --env DB_PARAMS --env SECRET_KEY --env LOGIN_DISABLED --env OAUTH_CLIENT_ID --env OAUTH_CLIENT_SECRET test

```

Comment:
You shouldn't need OAuth env vars in your tests when login is disabled.

Also, you could set things like LOGIN_DISABLED inline `--env LOGIN_DISABLED=True` rather than having to set it in Travis settings.